assets/layout.css

```diff
@@ -0,0 +1,618 @@
+hr {
+  height: 1px;
+  background: #ccc;
+}
+
+.alert {
+  height: 1rem;
+  margin: 0.5rem 1rem;
+  color: white;
+  padding: 0.8rem;
+  border-radius: 5px;
+}
+
+.alert-red {
+  background: red;
+}
+
+.alert-green {
+  background: green;
+}
+
+.grey {
+  background: #2b2a2a;
+}
+
+.quote-border-red {
+  border-left: 5px solid red;
+}
+
+.quote-border-green {
+  border-left: 5px solid green;
+}
+
+.quote-background-red {
+  background: #e50914;
+  color: #fff;
+}
+
+.quote-background-green {
+  background: green;
+}
+
+.quote-background-yellow {
+  background: yellow;
+}
+
+.quote-background-blue {
+  background: blue;
+  color: white;
+}
+
+.quote-background-orange {
+  background: orange;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.justify-content {
+  justify-content: space-between;
+}
+
+.border-none {
+  border: none;
+}
+
+.text-left {
+  float: left;
+}
+
+.text-right {
+  float: right;
+}
+
+.d-grid {
+  display: grid;
+}
+
+.grid-1 {
+  grid-template-columns: 1fr;
+}
+
+.grid-2 {
+  grid-template-columns: 1fr 1fr;
+}
+
+.grid-3 {
+  grid-template-columns: repeat(3, 1fr);
+}
+
+.d-block {
+  display: block;
+}
+
+.green {
+  color: rgb(13, 243, 13);
+}
+
+.red {
+  color: red;
+}
+
+.blue {
+  color: blue;
+}
+
+.white {
+  color: #fff;
+}
+
+.font-1 {
+  font-size: 1.02rem;
+}
+
+.font-2 {
+  font-size: 1.75rem;
+}
+
+.font-3 {
+  font-size: 2rem;
+}
+
+.li-none {
+  list-style: none;
+}
+
+.grid > div {
+  box-sizing: border-box;
+  float: left;
+}
+
+.grid.col-1 > div {
+  width: 100%;
+}
+
+.grid.col-2 > div {
+  width: 50%;
+}
+
+.grid.col-3 > div {
+  width: 33.33333%;
+}
+
+.grid.col-4 > div {
+  width: 25%;
+}
+
+.grid.col-5 > div {
+  width: 20%;
+}
+
+.grid.col-6 > div {
+  width: 16.66667%;
+}
+
+.grid.col-7 > div {
+  width: 14.28571%;
+}
+
+.grid.col-8 > div {
+  width: 12.5%;
+}
+
+.grid.col-9 > div {
+  width: 11.11111%;
+}
+
+.grid.col-10 > div {
+  width: 10%;
+}
+
+.grid.col-11 > div {
+  width: 9.09091%;
+}
+
+.grid.col-12 > div {
+  width: 8.33333%;
+}
+
+@media (max-width: 1024px) {
+  .ml-aa {
+    margin-left: 5%;
+  }
+
+  .display-lg-none {
+    display: none;
+  }
+
+  .grid.col-lg-1 > div {
+    width: 100%;
+  }
+
+  .grid.col-lg-2 > div {
+    width: 50%;
+  }
+
+  .grid.col-lg-3 > div {
+    width: 33.33333%;
+  }
+
+  .grid.col-lg-4 > div {
+    width: 25%;
+  }
+
+  .grid.col-lg-5 > div {
+    width: 20%;
+  }
+
+  .grid.col-lg-6 > div {
+    width: 16.66667%;
+  }
+
+  .grid.col-lg-7 > div {
+    width: 14.28571%;
+  }
+
+  .grid.col-lg-8 > div {
+    width: 12.5%;
+  }
+
+  .grid.col-lg-9 > div {
+    width: 11.11111%;
+  }
+
+  .grid.col-lg-10 > div {
+    width: 10%;
+  }
+
+  .grid.col-lg-11 > div {
+    width: 9.09091%;
+  }
+
+  .grid.col-lg-12 > div {
+    width: 8.33333%;
+  }
+
+  .clearfix {
+    grid-template-columns: 1fr !important;
+  }
+
+  img {
+    width: 100%;
+    margin: 0 auto;
+  }
+}
+
+@media (min-width: 720px) and (max-width: 1024px) {
+  .display-md-none {
+    display: none;
+  }
```

Comment:
Since you have this media query why did you define another one for `max-width: 1024px` just combine them because they do the same thing.